### PR TITLE
Added scale factor to minHeight logic

### LINKED ISSUE #29649: [Bug]: Stacked Bar chart is incorrectly represented when the data points on y-axis are larger than 1000
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz
    Memory: 28.48 GB / 47.71 GB
  Browsers:
    Edge: Chromium (118.0.2088.46)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://codepen.io/sobera123/pen/LYqVqRN

## Actual Behavior
When in a Stacked Vertical Bar Chart the values are high (> 1000), the bar heights are not matching the y-axis points on the left
image

## Expected Behavior
In the above case, 1st bar has total value of 2.5k and should have a height between the 2.9k and 3.9k references on y-axis

### Logs

_No response_

### Requested priority

High

### Products/sites affected

Internal Business Critical Product

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by Shubhabrata08**
## Previous Behavior

image

## New Behavior

image

## Related Issue(s)

- Fixes #29649

### CODE REVIEW

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx [hunk 1]
```diff
@@ -790,8 +790,8 @@ export class VerticalStackedBarChartBase extends React.Component<
         };
 
         let barHeight = heightValueScale * point.data;
-        if (barHeight < Math.max(Math.ceil(this._yMax / 100.0), barMinimumHeight)) {
-          barHeight = Math.max(Math.ceil(this._yMax / 100.0), barMinimumHeight);
+        if (barHeight < Math.max(heightValueScale * Math.ceil(this._yMax / 100.0), barMinimumHeight)) {
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx:793]: Add a change file for this change

Comment (Shubhabrata08) [reply packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx:793]: Adding asap

File: change/@fluentui-react-charting-558ce73b-bb6f-475f-be32-1aa78f0663a0.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added scale factor to minimum height logic",
```
Comment (AtishayMsft) [inline change/@fluentui-react-charting-558ce73b-bb6f-475f-be32-1aa78f0663a0.json:3]: In #29359 you also add logic to VerticalBarChart.
Does this fix apply to VerticalBarChart as well

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/25e02bde1d1de5a505c665eebb71b6885668939e'>25e02bde1d1de5a505c665eebb71b6885668939e</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-558ce73b-bb6f-475f-be32-1aa78f0663a0.json (hunk 2 above)
Comment (Shubhabrata08) [reply change/@fluentui-react-charting-558ce73b-bb6f-475f-be32-1aa78f0663a0.json:3]: image

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Pull request merged**